pkg/routeagent_driver/handlers/kubeproxy/vxlan.go

```diff
@@ -301,3 +304,25 @@ func (kp *SyncHandler) createVxLANInterface(activeEndPoint string, ifaceType int
 
 	return nil
 }
+
+func (kp *SyncHandler) ensureLooseModeIsConfigured(iface string) error {
+	for i := 0; i < 10; i++ {
+		// Revisit: This is a temporary work-around to fix https://github.com/submariner-io/submariner/issues/2422
+		// Allow the vx-submariner interface to get initialized.
+		time.Sleep(100 * time.Millisecond)
+
+		rpFilterSetting, err := kp.netLink.GetReversePathFilter(iface)
+		if err == nil {
+			if bytes.Equal(rpFilterSetting, []byte("2")) {
+				return nil
+			}
+		}
+
+		err = kp.netLink.EnableLooseModeReversePathFilter(iface)
+		if err != nil {
```

Comment:
Shouldn't we continue with the for loop instead of exiting in case of error here ?

Reply:
Failure to update the rp_filter setting on an interface was always logged and treated as an error. This PR preserves the existing behaviour.

Coming to the issue on RHEL-9, we do not get any error while updating the rp_filter, its just that it gets overwritten with some default config when a new interface is created.